pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -245,3 +245,5 @@ guiServer:
   port: 5683 # ENV: KUMA_GUI_SERVER_PORT
   # URL of the Api Server that requests with /api prefix will be redirected to. By default autoconfigured to http://locahost:port_of_api_server
   apiServerUrl: # ENV: KUMA_GUI_SERVER_API_SERVER_URL
+
+mode: standalone
```

Comment:
We need comment here explaining the option. Also, since this option is not configurable by ENV or config, please make a comment that this is configurable via arg